pkg/parser/parser.go

```diff
@@ -140,17 +140,21 @@ func dispatchArg(layerValue string, ima *v1.Image) {
 	if ima.Spec.ImageConfig.Args == nil {
 		ima.Spec.ImageConfig.Args = map[string]string{}
 	}
-	valueLine := strings.SplitN(layerValue, "=", 2)
-	if len(valueLine) != 2 {
-		logger.Error("invalid ARG value %s. ARG format must be key=value", layerValue)
-		return
-	}
-	k := strings.TrimSpace(valueLine[0])
-	if !utils.IsLetterOrNumber(k) {
-		logger.Error("ARG key must be letter or number,invalid ARG format will ignore this key %s.", k)
-		return
+
+	kv := strings.Split(layerValue, ",")
+	for _, element := range kv {
+		valueLine := strings.SplitN(element, "=", 2)
+		if len(valueLine) != 2 {
```

Comment:
What if value has "="     
like: runArg="--masters=xxx --nodes=xxx"

Reply:
if so, key="runArg" and value="--masters=xxx --nodes=xxx"。we only make sure that we got the the right key. the value is user set that is  meaningless for sealer build.